torchaudio/__init__.py

```diff
@@ -220,7 +246,9 @@ def info(filepath):
          >>> si, ei = torchaudio.info('foo.wav')
          >>> rate, channels, encoding = si.rate, si.channels, ei.encoding
      """
-    return _torch_sox.get_info(filepath)
+
+    info = getattr(_get_audio_backend_module(), 'info')
```

Comment:
nit: you can write this in one line ```getattr(_get_audio_backend_module(), 'info')(filepath)``` if you want.

Reply:
It is shorter, but is it more readable?